x/dsmr/certificate.go

```diff
@@ -89,13 +89,14 @@ func (c *ChunkCertificate) Verify(
 	if err != nil {
 		return fmt.Errorf("failed to initialize unsigned warp message: %w", err)
 	}
-
+	vs, err := warp.GetCanonicalValidatorSetFromChainID(ctx, pChainState, pChainHeight, msg.SourceChainID)
```

Comment:
Can we use a more descriptive variable name here, perhaps `canonicalValidatorSet` ?